.circleci/config.yml

```diff
@@ -377,14 +374,14 @@ commands:
     steps:
       - run:
           name: Install Python Dependencies with Pip
-          command: pip3 install --user -e . -r requirements.txt || pip3 install --user -e . -r requirements.txt
+          command: pip3 install --user -e . -r requirements.txt
```

Comment:
Some context of why the command was that way: https://github.com/nucypher/nucypher/issues/1482

Reply:
Hahaha :sweat_smile: